_test/page.getter.test.php

```diff
@@ -0,0 +1,32 @@
+<?php
+//======================================================================
+// PAGEGETTER TESTS
+//======================================================================
+
+require_once(__DIR__ . '/../DokuwikiXMLExport.php');
+require_once(__DIR__ . '/../admin.php');
+require_once(__DIR__ . '/../_test/Helper.php');
+require_once(__DIR__ . '/../PageGetter.php');
+
+class page_getter_test extends DokuWikiTest
```

Comment:
It is good practice that the filename and the class name should match, for example this will allow PHP to automatically discover in which file a class is to be found.